admin/client/src/components/PopoverTooltip/PopoverTooltip.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jQuery';
+import 'tetherWrapper';
+import 'bootstrap-popover';
+import 'bootstrap-tooltip';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class PopoverTooltip extends React.Component {
+  componentDidMount() {
+    const el = ReactDOM.findDOMNode(this);
+
+    // Enable BS Popovers and tooltips
+    $(el).find('[data-toggle="tooltip"]').tooltip();
+    $(el).find('[data-toggle="popover"]').popover({ html: true });
```

Comment:
I don't get it, how can this be both a `tooltip()` and a `popover()`? You can either have a `data-toggle="popover"` or a `data-toggle="tooltip"` attribute on the element. How do you see nested elements being both tooltips and popovers? Looking at the docs, we should be fine with just a popover? (http://v4-alpha.getbootstrap.com/components/tooltips/, http://v4-alpha.getbootstrap.com/components/popovers/).